splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/CreateTableWithDataIT.java

```diff
@@ -279,7 +279,7 @@ public void createTableWithViewJoins() throws Exception {
         String depsalTable = "depsal";
         String depsalTableRef = spliceSchemaWatcher.schemaName + "." + depsalTable;
         String depsalTableDef = format("create table %s as " +
-                "select dept, salary, ssn from %s with data", depsalTableRef, viewRef);
+                "select dept, salary, ssn from %s -- comment \n where dept < 3 with data", depsalTableRef, viewRef);
```

Comment:
Can you expand this case or add a second case where the embedded comment includes the word "with"?

Reply:
Done.